src/appleseed/renderer/modeling/project/project.xsd

```diff
@@ -302,17 +302,35 @@
                 </xsd:element>
                 <xsd:element name="output">
                     <xsd:complexType>
-                        <xsd:choice minOccurs="0" maxOccurs="unbounded">
+                        <xsd:all>
                             <xsd:element name="frame">
                                 <xsd:complexType>
                                     <xsd:complexContent>
                                         <xsd:extension base="parameterSet">
+                                            <xsd:choice minOccurs="0" maxOccurs="unbounded">

```

Comment:
Not sure here, maybe the maximum number of `<aovs>` element should be one? How many are we allowing elsewhere?